clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/UbamTests.scala

```diff
@@ -27,16 +29,25 @@ import org.scalatest.Assertion
 
 import scala.concurrent.Future
 
-/** Tests of Clio's wgs-ubam functionality. */
-trait WgsUbamTests { self: BaseIntegrationSpec =>
+/** Tests of Clio's ubam functionality. */
+trait UbamTests { self: BaseIntegrationSpec =>
 
-  def runUpsertWgsUbam(
-    key: TransferWgsUbamV1Key,
-    metadata: TransferWgsUbamV1Metadata
+  trait UbamType
```

Comment:
I'm guessing we're going to want this type in general. Could you move it to the `clio-util` subproject, and make it an enum? `Location` is an example of the setup.

Reply:
I'd also generalize `UbamType` to something like `SequencingType` (?).